eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,11 @@
+<!-- See aka.ms/dotnet/prebuilts for guidance on what pre-builts are and how to eliminate them. -->
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*" />
   </IgnorePatterns>
-</UsageData>
+  <Usages>
+    <!-- MSBuild versions starting from 16.10.0 do not target NET Core -->
+    <!-- and fail the non-source-build -->
```

Comment:
The exception in the main build is the following:

```
error NU1701: Package 'Microsoft.Build 16.10.0' was restored using '.NETFramework,Version=v4.6.1, ..., .NETFramework,Version=v4.8.1' instead of the project target framework '.NETCoreApp,Version=v3.1'. 
This package may not be fully compatible with your project.
```